src/translate.py

```diff
@@ -2741,16 +2767,20 @@ def function_abi(fn_sig: FunctionSignature, *, for_call: bool) -> Abi:
         only_floats = False
 
     for ind, param in enumerate(fn_sig.params):
-        size, align = param.type.get_size_align_bytes()
+        param_type = param.type
+        if param_type.is_array():
+            # Array parameters decay into pointers
```

Comment:
I wonder if a `.decay()` method on Type would be useful

Reply:
I had been going back-and-forth on this :+1: